UndertaleModTool/Editors/UndertaleSpriteEditor.xaml.cs

```diff
@@ -33,7 +33,7 @@ public UndertaleSpriteEditor()
 
         private void ExportAllSpine(SaveFileDialog dlg, UndertaleSprite sprite)
         {
-            MessageBox.Show("This seems to be a Spine sprite, .json and .atlas files will be exported together with the frames. " +
+            MessageBox.Show(Application.Current.MainWindow as MainWindow, "This seems to be a Spine sprite, .json and .atlas files will be exported together with the frames. " +
```

Comment:
Can you create a property for this, so it doesn't have to get copypasted to every catch?